HoI4Mod/Rome_Strikes_Back/events/tesb_ETH.txt

```diff
@@ -0,0 +1,2299 @@
+		trigger = {
+			NOT = { has_war_with = ETH }
+			is_in_faction = no
+		}
+		ai_chance = {
+			base = 100
+			modifier = {
+				factor = 0
+				NOT = { has_government = democratic }
+			}
+		}
+	}
+	# refuse
+	option = {
+		name = tesb_ETH.160.b
+		ETH = { country_event = { id = tesb_ETH.162 } }
+		ai_chance = {
+			base = 100
+			modifier = {
+				factor = 0
+				has_government = democratic
+			}
+		}
+	}
+}
+
+# Namibia accepts
+country_event = {
+	id = tesb_ETH.161
+	title = tesb_ETH.161.t
+	desc = tesb_ETH.161.d
+	picture = GFX_report_event_ger_civil_war
+
+	is_triggered_only = yes
+	option = {
+		name = tesb_ETH.161.a
+		add_to_faction = NMB
+	}
+}
+
+# Namibia refused to join
+country_event = {
+	id = tesb_ETH.162
+	title = tesb_ETH.162.t
+	desc = tesb_ETH.162.d
+	picture = GFX_report_event_ger_civil_war
+
+	is_triggered_only = yes
+	# allow them to refuse
+	option = {
+		name = tesb_ETH.162.a
+		ai_chance = {
+			base = 0
+		}
+	}
+	# force them to join
+	option = {
+		name = tesb_ETH.162.b
+		create_wargoal = {
+			type = puppet_wargoal_focus
+			target = NMB
+		}
+		ai_chance = {
+			base = 100
+		}
+	}
+}
+
+
+### Invite South Africa to faction events
+# Invite South Africa to faction
+country_event = {
+	id = tesb_ETH.170
+	title = tesb_ETH.170.t
+	desc = tesb_ETH.170.d
+	picture = GFX_report_event_ger_civil_war
+
+	is_triggered_only = yes
+	# accept unconditionally
+	option = {
+		name = tesb_ETH.170.a
+		ETH = { country_event = { id = tesb_ETH.171 } }
+		trigger = {
+			NOT = { has_war_with = ETH }
+			is_in_faction = no
+		}
+		ai_chance = {
+			base = 100
+			modifier = {
+				factor = 0
+				NOT = { has_government = democratic }
+			}
+		}
+	}
+	# refuse
+	option = {
+		name = tesb_ETH.170.b
+		ETH = { country_event = { id = tesb_ETH.172 } }
+		ai_chance = {
+			base = 100
+			modifier = {
+				factor = 0
+				has_government = democratic
+			}
+		}
+	}
+}
+
+# South Africa accepts
+country_event = {
+	id = tesb_ETH.171
+	title = tesb_ETH.171.t
+	desc = tesb_ETH.171.d
+	picture = GFX_report_event_ger_civil_war
+
+	is_triggered_only = yes
+	option = {
+		name = tesb_ETH.171.a
+		add_to_faction = SAF
+	}
+}
+
+# South Africa refused to join
+country_event = {
+	id = tesb_ETH.172
+	title = tesb_ETH.172.t
+	desc = tesb_ETH.172.d
+	picture = GFX_report_event_ger_civil_war
+
+	is_triggered_only = yes
+	# allow them to refuse
+	option = {
+		name = tesb_ETH.172.a
+		ai_chance = {
+			base = 0
+		}
+	}
+	# force them to join
+	option = {
+		name = tesb_ETH.172.b
+		create_wargoal = {
+			type = puppet_wargoal_focus
+			target = SAF
+		}
+		ai_chance = {
+			base = 100
+		}
+	}
+}
+
+
+### Invite Mutapa to faction events
+# Invite Mutapa to faction
+country_event = {
+	id = tesb_ETH.180
+	title = tesb_ETH.180.t
+	desc = tesb_ETH.180.d
+	picture = GFX_report_event_ger_civil_war
+
+	is_triggered_only = yes
+	# accept unconditionally
+	option = {
+		name = tesb_ETH.180.a
+		ETH = { country_event = { id = tesb_ETH.181 } }

```

Comment:
The simple answers (accept/refuse) can still go to the generic response events 11 and 12, so only the negotiation response needs bespoke events